tinyxml2.cpp

```diff
@@ -204,6 +205,7 @@ char* StrPair::ParseText( char* p, const char* endTag, int strFlags )
             return p + length;
         }
         ++p;
+        TIXMLASSERT( p );

```

Comment:
@leethomason It's not only unlikely, it's next to impossible. The assertion is to better illustrate what to expect from the code.